cypress/integration/core-opensearch-dashboards/opensearch-dashboards/datasource-management-plugin/1_create_datasource.spec.js

```diff
@@ -104,11 +104,11 @@ if (Cypress.env('DATASOURCE_MANAGEMENT_ENABLED')) {
           'app/management/opensearch-dashboards/dataSources'
         );
       });
-
-      it('with sigV4 and all required inputs to connect to OpenSearch Service', () => {
+      // TODO: once create datasource with sigv4 is in plance, remove the skip
```

Comment:
typo?

Reply:
I added that on purpose, since current test infra in workflow was added to support no auth and basic auth. It doesn't support creating AOS/AOSS domain yet. We need to think of a way to either creating one from infra, or add a long running one by configuring it in github secrets for testing purpose

Reply:
can add more env variables too whether to ran specific suite of tests